_posts/2018-08-09-deploying-esxi-through-foreman.md

```diff
@@ -88,7 +91,7 @@ Note: This demo is run on a Ubuntu system. The paths may be different for other
       NOHALT 1
       LABEL ESXi
       KERNEL ../boot/ESXi-6.7.0-8169922/mboot.c32
-      APPEND -c ../boot-ESXi-6.7.0-8169922.cfg
+      APPEND -c ../boot-ESXi-6.7.0-8169922.cfg ks=<%= foreman_url("provision") %>
```

Comment:
Added kickstart kernel param in the PXELinux template directly to fetch provision link **with token**!